api-1.9/pom.xml

```diff
@@ -35,21 +36,40 @@
 		<dependency>
 			<groupId>org.openmrs.api</groupId>
 			<artifactId>openmrs-api</artifactId>
-			<version>${openmrs.1.9.api.version}</version><!--$NO-MVN-MAN-VER$-->
+			<version>${openmrs.1.9.api.version}</version>
+			<!--$NO-MVN-MAN-VER$-->
 		</dependency>
-
+		
 		<dependency>
 			<groupId>org.openmrs.api</groupId>
 			<artifactId>openmrs-api</artifactId>
-			<version>${openmrs.1.9.api.version}</version><!--$NO-MVN-MAN-VER$-->
+			<version>${openmrs.1.9.api.version}</version>
+			<!--$NO-MVN-MAN-VER$-->
 			<classifier>tests</classifier>
 		</dependency>
 		
 		<dependency>
 			<groupId>org.openmrs.test</groupId>
 			<artifactId>openmrs-test</artifactId>
-			<version>${openmrs.1.9.api.version}</version><!--$NO-MVN-MAN-VER$-->
+			<version>${openmrs.1.9.api.version}</version>
+			<!--$NO-MVN-MAN-VER$-->
 			<type>pom</type>
 		</dependency>
 	</dependencies>
+	
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>com.googlecode.maven-java-formatter-plugin</groupId>

```

Comment:
I'm surprised you need to add it to a pom of every submodule. I think it should be enough to have it in the parent pom only.